docs/topics/mta-web-configuring-and-running-an-application-analysis.adoc

```diff
@@ -92,6 +92,17 @@ Analysis takes minutes to hours to run depending on the size of the application
 ====
 {ProductShortName} relies on Kubernetes scheduling capabilities to determine how many analyzer instances are created based on cluster capacity. If several applications are selected for analysis, by default, only one analyzer can be provisioned at a time. With more cluster capacity, more analysis processes can be executed in parallel.
 ====
+
+. Optional: To track the status of your active analysis tasks, click the *Task Manager* tab or the popover message on the application name. The *Task Manager* displays the following information:
```

Comment:
Optional: To track the status of your active analysis tasks, open the *Task Manager drawer* by clicking the notification badge or the popover message on the application name

No need to mention the Task manager table here, IMO